docs/common/craft-parts/reference/plugins/ant_plugin.rst

```diff
@@ -4,6 +4,8 @@ Ant plugin
 ==========
 
 The Ant plugin builds Java projects using the `Apache Ant`_ build tool.
+This plugin will set the ``JAVA_HOME`` environment variable to the
+path to the latest JDK found in the build environment.
```

Comment:
This probably needs an explanation of the `ANT_OPTS`environment variable that gets set too.